automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceSpec.scala

```diff
@@ -313,9 +313,9 @@ class WorkspaceSpec extends FreeSpec with WebBrowserSpec with WorkspaceFixtures
             val detailPage = listPage.enterWorkspace(billingProject, workspaceName)
             val methodConfigTab = detailPage.goToMethodConfigTab()
             val methodConfigDetailsPage = methodConfigTab.openMethodConfig(SimpleMethodConfig.configNamespace, s"$methodConfigName")
-            val errorModal = methodConfigDetailsPage.clickLaunchAnalysisButtonError()
-            errorModal.getErrorText shouldBe "You do not have access to run analysis.\nCancel"
-            errorModal.clickCancel()
+            val msgModal = methodConfigDetailsPage.clickLaunchAnalysisButtonMessage()
+            msgModal.getMessageText shouldBe "You do not have access to run analysis.\nCancel"
```

Comment:
We should set this message as a "constant" (either in the error modal or at the class level here) and validate it against that. The other option is to create a method in errorModal to handle validating the error message and to call that.